specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobPrivateEndpoints.json

```diff
@@ -0,0 +1,337 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/jobAgents/{jobAgentName}/privateEndpoints": {
+      "get": {
+        "tags": [
+          "JobPrivateEndpoints"
+        ],
+        "description": "Gets a list of job agent private endpoints.",
+        "operationId": "JobPrivateEndpoints_ListByAgent",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "jobAgentName",
+            "in": "path",
+            "description": "The name of the job agent.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of private endpoints.",
+            "schema": {
+              "$ref": "#/definitions/JobPrivateEndpointListResult"
+            }
+          },
+          "default": {
```

Comment:
Consider use `../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse`, please also update other places

Reply:
Unfortunately, SQL generators do not have those common types available. We have already filed a task to align the SQL common types. Other reviewers agreed that this is sufficient for now as this would be a massive effort that can't be done in this one API version. https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/2697929